### PR TITLE
update bundler and nokogiri

### PR DESCRIPTION
**Pull request by AndrewHaisting**

### CODE REVIEW

**Review by shadyoak — APPROVED** [review]
👨‍🍳💋

File: Gemfile.lock [hunk 1]
```diff
@@ -72,7 +72,7 @@ GEM
     nap (1.1.0)
     nenv (0.3.0)
     no_proxy_fix (0.1.2)
-    nokogiri (1.10.9)
+    nokogiri (1.10.10)
```
Comment (dperez37) [inline Gemfile.lock:75]: Not going all the way to `1.11.1`?

Comment (AndrewHaisting) [reply Gemfile.lock:75]: Nah, I just run bundle update :)

Comment (dperez37) [reply Gemfile.lock:75]: Fair enough!

Comment (shadyoak) [reply Gemfile.lock:75]: RELEASE THE BOTS

**Pull request merged**